ui/src/testing/stubs/cluster-api-stub.ts

```diff
@@ -62,4 +63,11 @@ export class ClusterApiStub extends ClusterApi {
       resolve(cdrVersionList);
     });
   }
+
+  localize(projectName: string, clusterName: string, req: ClusterLocalizeRequest,
+    extraHttpRequestParams?: any): Promise<ClusterLocalizeResponse> {
+    return new Promise<ClusterLocalizeResponse>(resolve => {
+      resolve({clusterLocalDirectory: 'workspaces/${req.workspaceId}'});
```

Comment:
nit: can we put a function to generate the local directory on either ClusterRequest or some other handy place? This particular function may not be the place for this formatting knowledge (unless it really is the only place we need to do this).